src/licensedcode/data/licenses/ecfonts-1.0.yml

```diff
@@ -0,0 +1,7 @@
+key: ecfonts-1.0
+shortname: latex-ec-fonts
```

Comment:
this should be short_name

Reply:
Apologies for the late follow-up. I have taken a note of the mistake and rectified it by editing the file. Kindly take a look and I'll keep a watch for any more tests failing. Also I need a bit of help from you for future contributions : how do I understand the possible cause and location of the file for a failing test from the logs. I think a bit of guidance in this regard can help me contribute better in my future commits. Thanks!